camel/types/enums.py

```diff
@@ -265,6 +269,8 @@ def token_limit(self) -> int:
             ModelType.MISTRAL_LARGE,
             ModelType.MISTRAL_NEMO,
             ModelType.MISTRAL_PIXTRAL_12B,
+            ModelType.MISTRAL_8B,
+            ModelType.MISTRAL_3B,
```

Comment:
```suggestion
            ModelType.MISTRAL_MINISTRAL_3B,
            ModelType.MISTRAL_MINISTRAL_8B,
```

Reply:
Sorry it seems to fine to ccall `ModelType.MISTRAL_3B`. Can we move it up and sort by the names?

Reply:
updated